src/js/state.js

```diff
@@ -41,6 +41,12 @@ export default async function () {
   } else {
     ciphers = ciphers;
   }
+  if (configs[server].usesOpenssl && minver('3.0.0', form['openssl'].value)) {
+    // set SECLEVEL=0 via cipher string to support TLSv1-1.1 "old" with OpenSSL 3.x
+    if (protocols.includes('TLSv1.1')) {
+      if (!ciphers.includes('@SECLEVEL=0')) ciphers.unshift('@SECLEVEL=0');
+    }
+  }
```

Comment:
Looking at line lengths around (as L34) with similar conditionals, there's no formatting enforced really, but the order of evaluation mostly preferred, so I went for speed and simplification and e.g. this one stops evaluating once the leftmost is not relevant, so the protocol should go first etc.:
 
```suggestion
  if (protocols.includes('TLSv1.1') && configs[server].usesOpenssl && minver('3.0.0', form['openssl'].value)) {
    // to allow "old" TLSv1-1.1 with OpenSSL 3.x we add SECLEVEL=0 via cipher string
    if (!ciphers.includes('@SECLEVEL=0') && ciphers.length) ciphers.unshift('@SECLEVEL=0');
  }
```

Also checking that we're actually adding it to an existing list, not creating a bogus list from empty result (there may not necessarily be an overlap of the guideline and the actual server able to offer own list of ciphers)

Reply:
Ah, with the defaults not set in configs, the usesOpenssl value is not truthy, it's just unset, so needs precise testing… sorry.

```suggestion
  if (protocols.includes('TLSv1.1') && configs[server].usesOpenssl !== false && minver('3.0.0', form['openssl'].value)) {
    // to allow "old" TLSv1-1.1 with OpenSSL 3.x we add SECLEVEL=0 via cipher string
    if (!ciphers.includes('@SECLEVEL=0') && ciphers.length) ciphers.unshift('@SECLEVEL=0');
  }
```

Reply:
> Ah, with the defaults not set in configs, the usesOpenssl value is not truthy, it's just unset, so needs precise testing… sorry.

Ok.  Then your changes should have been minimal changes to address that, not rewriting other parts (yet again).  Please try to be much, MUCH, more precise in making the minimum necessary changes.

I wrote the conditional logic in a specific order for a reason.  The check for TLSv1.1 in protocols is only necessary for openssl 3.x where we have to change the security level.  You checking for TLSv1.1 in protocols first "because you wanted to" is not the logical progression that I communicated in the way that I coded it or the intention of this PR, which is titled "Updates to support OpenSSL 3.x".  Your change to reorder the logic was unnecessary and poor code review hygiene, besides a waste of time for all involved.  Please try to get things done more expediently.

Reply:
The review suggestion was not addressed so I went on to make the changes myself to finish testing and be able to release this.

If OpenSSL 3.x is gonna be the default, it will always be that case, so doesn't make sense to test first, the earliest to bail is to test for protocols, don't understand the rest of your concerns (whether they are real or just hypothetical, you'd have to be more specific, sorry.)

If you don't like my reviews, don't request them. Thanks. It was not in a working state i.e. not even tested.

Reply:
Reasoning for order was in the first comment of this thread above the suggestion. Not "because I wanted to".

Reasoning for not creating just "@SECLEVEL=0" cipherstring from an empty list due to possible `configs[server].supportedCiphers` intersection was described in the first post below the code suggestion.

Your rebase removing these, on the other hand, doesn't seem to be explained anywhere; so I'll just leave it to the others — as indeed, going back for the third time un-solving the comments at any cost really is a waste of everyone's time.